tensorflow/lite/micro/kernels/xtensa/lstm/helper_utils/portable_tensor_utils.h

```diff
@@ -0,0 +1,320 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
```

Comment:
As above, is this a copied file?

Reply:
lstm uses some functions from this file

Reply:
From the email discussion with @advaitjain, I'm wondering if you still need these copies of TFLite files, or if you can use the originals now? I think this applies to portable_tensor_utils.*, tensor_utils.*, and util.*

Reply:
@petewarden 
Completely removed util.cc and made few changes from the original file of portable_tensor_utils.cc to fix compiler warnings and if you are okay I like to keep it in xtensa/lstm/helper_functions/portable_tensor_utils.,tensor_utils. for sometime and later I will replace our file with @advaitjain PR which will bring these files from Tflite impl.